{"uuid": "7340735c-648e-486b-b669-0a2ebe35cd38", "children": ["4190f5cb-d579-4f83-93f8-ae91b332bf6e"], "befores": [{"name": "chrome_driver_init", "status": "broken", "statusDetails": {"message": "browsermobproxy.exceptions.ProxyServerError: The Browsermob-Proxy server process failed to start. Check <_io.TextIOWrapper name='D:\\\\python-project\\\\webautotest\\\\server.log' mode='w' encoding='cp936'>for a helpful error message.\n", "trace": "  File \"E:\\Python3.11\\Lib\\site-packages\\pluggy\\_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\Python3.11\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\Python3.11\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\python-project\\webautotest\\bbs\\conftest.py\", line 19, in chrome_driver_init\n    proxy, server = start_browsermob_proxy(config.BROWSERMOB_PROXY_PATH, port=config.BROWSERMOB_PROXY_LOCAL_PORT)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\python-project\\webautotest\\bbs\\common\\browsermob_proxy.py\", line 11, in start_browsermob_proxy\n    server.start()\n  File \"E:\\Python3.11\\Lib\\site-packages\\browsermobproxy\\server.py\", line 122, in start\n    raise ProxyServerError(message)\n"}, "start": 1691487065497, "stop": 1691487068625}], "start": 1691487065497, "stop": 1691487069172}